### ISSUE TITLE
Any way to incorporate MipGenSettings?

### ISSUE THREAD

**Issue by Ident** [hollowdilnik/filtered-widget#2]
I tried changing the MipGenSettings on the RenderTarget but it had no effect at all. I guess autogenerate mipmaps uses a default way (averaging?) for creating mipmaps and that's it. Is that right?

**Comment by da-molchanov** [comment]
I don't think you have any control over the way mips are generating at runtime, Looking at `ComputeGenerateMips.usf`, I think it is just downsampling the texture without even averaging it?
